### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] N/A
[Problem] No contribution guideline.
[Solution] Add contributing rules

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+how problem was fixed.
+
+[Remarks] Optional section containing additional comments on change, its
+connection to other repositories, announcement of forthcoming changes related
+to issue, etc.
+
+[Test] Optional section describing how to test change after applying it. Added
+if it deviates from currently documented test methods.
+
+Signed-off-by: info added when `git commit` ran wih `-s` option
```
Comment (TomaszLukawskiSam) [inline CONTRIBUTING.md]: wih -> with

Comment (lslachciak) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+### Coding styles
+Project has defined eslint rules (.eslintrc.js)
+We recommend to run "grunt eslint" before uploading a pull request.
+
+### Review policy
+Assign Reviewer in Reviewers panel.
+
+PR can be merged by TAU Reviewers only.
+
+##### TAU reviewers:
```
Comment (m-niesluchow) [inline CONTRIBUTING.md]: How about adding to CODEOWNERS instead?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Contributing guidelines
+
+### Contributing code
+If you have improvements to TAU, send us your pull requests! For those just getting started, Github has a [howto](https://help.github.com/articles/using-pull-requests/).
+
+### Commit policy
+We recommend that use a 1 commit for the pull request.
+
+### Commit message
```
Comment (m-niesluchow) [inline CONTRIBUTING.md]: When viewed ([1]) lines in commit message are not broken. Line size restrictions?

[1] https://github.com/Samsung/TAU/blob/8c4efcd7cb0d59022319f0eafe1f621f60dce96e/CONTRIBUTING.md

File: CONTRIBUTING.md (hunk 2 above)
Comment (lslachciak) [reply CONTRIBUTING.md]: not yet, let's do this later

**Pull request merged**